recipe/meta.yaml

```diff
@@ -1,44 +1,59 @@
 {% set version = "0.5.4" %}
+{% set name = "imutils" %}
 
 package:
-  name: imutils
+  name: {{ name }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/i/imutils/imutils-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: 03827a9fca8b5c540305c0844a62591cf35a0caec199cb0f2f0a4a0fb15d8f24
 
 build:
-  number: 3
-  script: '{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv '
+  number: 0
+  # No opencv on linux-s390x
+  skip: true  # [linux and s390x]
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-build-isolation -vv 
```

Comment:
I wonder if `--ignore-installed` should be added by default or not. The preference in this case is a little ambiguous to me (unlike `--no-deps` and `--no-build-isolation`)

Reply:
--ignore-installed flag, as far as I am aware, forces pip install to disregard any installed versions of the current package that might already be installed and proceed with installing the specified version. I don't see why wee need this flag as we are typically building in a clean environment.

Reply:
That's right. That's why I said that our preference for this flag is ambiguous to me (either we should always prefer it, or we should always deprecate it). Maybe I would remove it, or get a second opinion. Up to you.

Reply:
I think I left it as it was in CF. I don't think it makes a difference. Let's remove it.